### PR TITLE
Update ingest config example dataset

### PR DESCRIPTION
**Pull request by adurkan-amido**
Update example ingest pipelines to use movies dataset.
ADF pipeline amendments to update output path (also updated in jinja template)
Fix to E2E test utils to ensure only the test directory is checked for output files

### CODE REVIEW

File: de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json [hunk 1]
```diff
@@ -1,40 +1,51 @@
...
-            "schema": "SalesLT",
-            "table": "SalesOrderDetail",
-            "columns": "SalesOrderID, SalesOrderDetailID, OrderQty, ProductID, UnitPrice, UnitPriceDiscount, LineTotal, rowguid, ModifiedDate",
+            "schema": "movies",
+            "table": "ratings_small",
+            "columns": "[userId], [movieId], [rating], [timestamp]",
             "load_type": "delta",
-            "delta_date_column": "ModifiedDate",
-            "delta_upsert_key": "SalesOrderID, SalesOrderDetailID"
+            "delta_date_column": "DATEADD(MINUTE,[timestamp]/60,'1970-01-01 00:00:00.0')",
```
Comment (motylp) [inline de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json:47]: this feels too complicated, could you try using [dateTimeFromEpoch](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-date#datetimefromepoch)?

File: de_templates/ingest/Ingest_SourceType_SourceName/data_factory/pipelines/ARM_IngestTemplate.json.jinja [hunk 2]
```diff
@@ -199,7 +205,48 @@
                                             }
                                         }
                                     }
-                                }
+                                },
+                                {
+                        "name": "correlation_id",
```
Comment (motylp) [inline de_templates/ingest/Ingest_SourceType_SourceName/data_factory/pipelines/ARM_IngestTemplate.json.jinja:210]: I think this correlation_id is quite confusing for someone not very technical looking at the pipelines. Maybe pipeline_id would be more suitable? Feel free to resolve though.

File: de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json (hunk 1 above)
Comment (adurkan-amido) [reply de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json:47]: The reason is that this needs to run dynamically on the data source itself, so this is T-SQL and will get used in the query, e.g.
```
SELECT [userId], [movieId], [rating], [timestamp]
FROM movies.ratings_small
WHERE DATEADD(MINUTE,[timestamp]/60,'1970-01-01 00:00:00.0') >= '2010-01-01'
AND DATEADD(MINUTE,[timestamp]/60,'1970-01-01 00:00:00.0') < '2010-01-31';
```

Comment (motylp) [reply de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json:47]: OK, I understand, looks like `DATEADD` is the way then. But do we really need to do this extra calculations for minutes? Why not simply `DATEADD(SECOND,[timestamp],'19700101')`?

File: de_templates/ingest/Ingest_SourceType_SourceName/data_factory/pipelines/ARM_IngestTemplate.json.jinja (hunk 2 above)
Comment (adurkan-amido) [reply de_templates/ingest/Ingest_SourceType_SourceName/data_factory/pipelines/ARM_IngestTemplate.json.jinja]: Changed to `run_id` throughout

File: de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json (hunk 1 above)
Comment (adurkan-amido) [reply de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json]: Good point, updated

**Review by motylp — APPROVED** [review]

**Review by motylp — APPROVED** [review]

**Review by motylp — APPROVED** [review]

**Pull request merged**